tests/helper/test_helper.py

```diff
@@ -151,25 +151,23 @@ def test_index_starred(
     assert index_messages(messages, model, model.index) == expected_index
 
 
-@pytest.mark.parametrize(
-    "mentioned_messages",
-    [
-        {537286, 537287, 537288},
-        {537286},
-        {537287},
-        {537288},
-        {537286, 537287},
-        {537286, 537288},
-        {537287, 537288},
-    ],
-)
```

Comment:
It'd be clearer to sweep all of these in the first pass as a direct refactor wherever these are used, then you can expand it after that. As it is it makes it more difficult to compare this removed text with that removed/swapped in the previous commit. Refactoring first and then switching the fixture second should make this clearer.